README.md

```diff
@@ -39,15 +39,17 @@ export DOMAIN_NAME=prombench.prometheus.io
 ```
 ./prombench gke resource apply -a $AUTH_FILE -v PROJECT_ID:$PROJECT_ID -v ZONE:$ZONE \
     -v CLUSTER_NAME:$CLUSTER_NAME -v GCLOUD_SERVICEACCOUNT_CLIENT_EMAIL:$GCLOUD_SERVICEACCOUNT_CLIENT_EMAIL \
-    -f components/prow/manifests/rbac.yaml -f components/prow/manifests/nginx-controller.yaml
+    -f manifests/cluster-wide
 ```
 
+This will be good time to set the A record for `<DOMAIN_NAME>` to the nginx-ingress-controller IP address. See [Deploy Prometheus-Meta & Grafana](#deploy-prometheus-meta--grafana-1).
```

Comment:
instead of this can just move 	this section here
https://github.com/prometheus/prombench/pull/201/files#diff-04c6e90faac2675aa89e2176d2eec7d8L147

btw is using a domain optional? If someone wants to run a quick test without setting a domain name, is this possible?

Reply:
The services will be available in the ingress IP but since `--web.external-url` flag is used for prometheus and the domain name is also used in the grafana data source, some parts will not function properly without setting up the domain pointing to the ingress IP.

not having a domain point to the ingress IP will not break the deployment but all functionalities will not function as expected.